docs/source/advanced/test-builders.mdx

```diff
@@ -1,38 +1,32 @@
 ---
-title: 'Test Builders'
+title: Test builders
```

Comment:
How do we feel about "test builders" not automatically being capitalized? We try to err on the side of not doing it except for titled features and products

Reply:
👍 
We were wondering about that - that answers it :)

Reply:
Ahah yes we had that same question. I'd go further and ask whether this can be considered a titled feature or not :)

Reply:
I would lean toward no for simplicity's sake, given that it's pretty rare that we capitalize (for example, "schema checks" is lowercase, and those are a pretty big deal in Studio)

Reply:
Test builders are definitely a less prominent feature than schema checks 👍